docs/detections/alerts-ui-manage.asciidoc

```diff
@@ -161,6 +162,23 @@ image::images/alert-change-status.png[Bulk action menu with multiple alerts sele
 
 * In an alert's details flyout, click *Take action* and select a status.
 
+[float]
+[[apply-alert-tags]]
+==== Apply and filter alert tags
+
+Use alert tags to organize related alerts into categories that you can filter and group. For example, use the `False Positive` alert tag to label a group of alerts as false positives. Then, search for them by entering the `kibana.alert.workflow_tags : "False Positive"` query to the KQL bar.  
+
+NOTE: You can manage alert tag options by updating the `securitySolution:alertTags` advanced setting. Refer to <<manage-alert-tags>> for more information. 
+
+To apply or remove alert tags on individual alerts, do one of the following:
+
+** In the Alerts table, click *More actions* (*...*) in the alert's row, then click *Apply alert tags*. Select or unselect tags, then click *Apply tags*. 
+** In an alert’s details flyout, click *Apply alert tags*. Select or unselect tags, then click *Apply tags*. 
```

Comment:
Would they need to select *Take action* first?

Reply:
Oops, accidentally deleted that part of the step. Thanks!

```suggestion
** In an alert’s details flyout, click *Take action -> Apply alert tags*. Select or unselect tags, then click *Apply tags*. 
```